packages/apollos-data-prayer/src/prayer-requests/__tests__/resolver.test.js

```diff
@@ -49,7 +49,7 @@ describe('PrayerRequest resolvers', () => {
   it('gets all public prayer requests', async () => {
     const query = `
       query {
-        getPublicPrayerRequests {
+        prayers {
```

Comment:
It might be important to insert the `public` into this namespace.

Reply:
I thought about that but since *all* we deal with is public prayers it makes sense to just have cleaner code. if in the future we decide to do something with private prayers, we can call those `privatePrayers`.